src/app/components/util/chpl-sortable-headers.jsx

```diff
@@ -87,9 +95,11 @@ ChplSortableHeaders.propTypes = {
     text: string,
     sortable: bool,
   })).isRequired,
+  stickyHeader: bool,
 };
 
 ChplSortableHeaders.defaultProps = {
   order: '',
   orderBy: '',
-};
+  stickyHeader: false,
+};
```

Comment:
There should be a new line after this one, as in, you should go to the end of this line, hit "Enter", and save that